auth/only_api_test.go

```diff
@@ -1,404 +0,0 @@
-package auth
```

Comment:
Can we still test this?

Reply:
Gonna bump this. It seems correct for now, but I don't want someone to change something later and it breaks all our service users.

Reply:
Oops, restored this and removed the tests that are updating data